src/Geboortezorg-3-2/Cert/PHR-Client/gebz-phr-1-5-integrale-zwangerschapskaart.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://hl7.org/fhir/STU3/testscript.sch" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TestScript xmlns="http://hl7.org/fhir" xmlns:nts="http://nictiz.nl/xsl/testscript" nts:scenario="client">
+   <id value="gbz-fhir3-0-2-ikz-phr-1-4"/>
+   <version value="stu3-3.0"/>
+   <name value="Nictiz Geboortezorg - Client - Scenario 1.5 - Retrieve Integrale Zwangerschapskaart of Julia Laarhoven (no data available)"/>
+   <description value="Scenario 1.5: Retrieve Integrale Zwangerschapskaart of Julia Laarhoven (no data available)"/>
+   <nts:authToken patientResourceId="JULIAJXXX-LAARHOVEN"/>
+   
+   <!-- Patient - Zwangere vrouw -->
+   <test id="1-RetrieveWoman">
+      <name value="1-RetrieveWoman"/>
+      <description value="Query Patient resource for HCIM Patient"/>
+      <nts:include value="gbz-operation-search-phr">
```

Comment:
Je zou nog kunnen overwegen die assert.request.authorizationHeader, canary-assert en assert.response.numResources nog in gbz-operation-search kunnen trekken.

Reply:
opgelost